bin/build_openmp.sh

```diff
@@ -226,6 +255,17 @@ if [ "$1" == "install" ] ; then
       exit 1
    fi
 
+   if [ "$AOMP_BUILD_DEVEL" == "1" ]; then
+     cd $BUILD_DIR/build/openmp_devel
+     echo
+     echo " -----Installing to $INSTALL_OPENMP/lib-devel ----- "
+     $SUDO $AOMP_NINJA_BIN -j $AOMP_JOB_THREADS install
```

Comment:
Should we consider installing into a separate lib-dev instead of lib, like we have lib-debug?  Then it is possible to switch between the performance and dev versions...like we can with lib-debug:

$ LD_LIBRARY_PATH=$AOMP/lib-debug rocgdb ./helloworld

Reply:
This is indeed the case. We set the `LLVM_LIBDIR_PATH` in line 140 on this patch to the new `lib-devel` directory.